test/e2e/operator.go

```diff
@@ -391,7 +379,7 @@ var _ = Describe("ARO Operator - Azure Subnet Reconciler", func() {
 	}
 
 	BeforeEach(func() {
-		enableReconcileSubnet()
+		//enableReconcileSubnet() // already enabled in the PR
```

Comment:
you can remove this comment and return it to just being deleted

Reply:
Removed.